examples/migrate-pvc/README.MD

```diff
@@ -0,0 +1,125 @@
+# Migrate a PVC from a kubernetes cluster to another one 
+
+We'll demonstrate how to migrate a PVC from a cluster (source cluster )
+to another one (destination cluster) with Kanister.
+
+This example was tested to migrate a PVC from an openshift 3.9 cluster to an openshift 4.7 cluster.
+It's for this reason we install kanister in version 0.61.0 because it's the last kanister version still 
+compatible with openshift 3.9. If you don't have this constraint you can install kanister in the latest
+version.
+
+## Requirement 
+
+The PVC must not be mounted by any pods during the backup or during the restore.
+
+## Prerequisite  
+
+
+### Install kanister on both cluster 
+
+```
+helm install myrelease --namespace kanister kanister/kanister-operator --set image.tag=0.61.0 --version 0.61.0
+```
+
+### Install the same profile and blueprint on both cluster 
+
+```
+kubectl create -f copy-volume-data-bp.yaml
+
+helm install profile kanister/profile \
+    --namespace kanister \
+    --version 0.61.0 \
+    --set defaultProfile=true \
+    --set location.type='s3Compliant' \
+    --set location.bucket="<MY_BUCKET_NAME>" \
+    --set location.region="<MY_AWS_REGION>" \
+    --set aws.accessKey="<MY_AWS_ACCESS_KEY>" \
+    --set aws.secretKey="<MY_AWS_SECRET_ACCESS_KEY>"
+```
+
+### Install kanctl 
+
+Install [kanctl](https://github.com/kanisterio/kanister/releases/tag/0.61.0), it's a simple executable.
+Add it to your path.
+
+
+## Create a PVC in a namespace 
+
+Switch your kubectl context to your source cluster.
+
+Create the test-pvc ns and a PVC with data inside
+```
+kubectl create ns test-pvc
+kubectl create -f my-pvc.yaml -n test-pvc
+kubectl create -f create-data.yaml -n test-pvc
+kubectl create -f explore-pvc.yaml -n test-pvc
+```
+
+Use explore-data to explore the data in the PVC 
+```
+kubectl exec -it -n test-pvc explore-data -- /bin/sh
+/ # ls /my-pvc/
+my-file
+/ # cat my-pvc/my-file 
+hello world
+/ # exit 
+```
+
+Delete the pod explore-data to free the PVC.
+```
+kubectl delete -f explore-pvc.yaml
+```
+
+## Run a backup of the PVC with kanctl 
+
+### Create a backup 
+
+Use kanctl to create the backup 
+```
+kanctl create actionset -n kanister -a backup -v test-pvc/my-pvc -b copy-volume-bp -p default-profile
+actionset backup-mvwhh created
+```
+
+Check the status of the actionset 
+```
+kubectl get actionset backup-mvwhh -n kanister -o json | jq '.status.state'
+"complete"
+```
+
+PVC content has been sent to the S3 bucket defined in default-profile.
+
+### Prepare a restore-action for the destination cluster.
+
+Create a dry-run restore action that will execute in the destination cluster.
+
+```
+kanctl --namespace kanister create actionset -a restore --from backup-mvwhh --dry-run > restore-backup-mvwhh.yaml
```

Comment:
No that won't work, because this action set is built from an existing action set in the source cluster. Once the user have changed his context to the destination cluster it won't be able to generate it anymore as this "from" action set does not exist in the destination cluster.